newArrheniusEq.jl

```diff
@@ -0,0 +1,56 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .jl
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.16.1
+#   kernelspec:
+#     display_name: Python 3
+#     name: python3
+# ---
+
+# + id="OSOaG-m77vLy"
+#include("Energies.jl")
+
+kb = 8.617e-5 #eV/K
+"""
+function flow_coefficient(type, effecting_layers, decay_constant)
+    A = 1.0
+    if type == "exponential"
+        return A * exp(-decay_constant * coveringlayers)
+    elseif type == "linear"
+        return A * (1 - decay_constant * coveringlayers)
+    end
+end
+"""
+
+function flow_coefficient(type, effecting_nums, decay_coefficients)

```

Comment:
write this for a single value of the decay coefficient for easier readability, as I said above, can either define a custom broadcast if needed to work on an array, or just do that as a list comprehension later